src/components/SystemPreferences.vue

```diff
@@ -179,6 +188,26 @@ export default {
         @change="updatedCPU"
       />
     </div>
+    <hr>
+
+    <div class="checkbox">
+      <Checkbox
+        :value="telemetry"
+        label="Allow collection of anonymous statistics to help us improve Rancher Desktop"
+        @input="toggleTelemetry"
+      />
+      <v-popover placement="right">
+        <i class="icon icon-info" />
+        <span slot="popover">
+          Rancher Labs would like to collect a bit of anonymized information<br />
+          about the configuration of your installation to help make Rancher Desktop better.<br /><br />
+          Your data will not be shared with anyone else, and no information about<br />
+          what specific resources or endpoints you are deploying is included.<br />
+          Once enabled you can view exactly what data will be sent at <code>/v1-telemetry</code>.<br /><br />
+          <a href="https://rancher.com/docs/rancher/v2.x/en/faq/telemetry/" target="_blank">More Info</a>
```

Comment:
This needs to open in the user's [default browser](https://www.electronjs.org/docs/api/shell#shellopenexternalurl-options), _not_ in Electron.  We may need to do that in the main process on [an event](https://www.electronjs.org/docs/api/web-contents#event-will-navigate) for all `http` links.

Reply:
This code was dropped.